spec/text_spec.rb

```diff
@@ -0,0 +1,27 @@
+require 'text'
+
+describe Text do
+  let(:input_message) {"hello can i have a menu"}
+  let(:reply) {double(:reply)}
+  let(:message) {"hello can i have a menu"}
+  let(:client) {double(:client)}
+  subject(:text) {described_class.new client}
+
+
+  before(:example) do
+    allow_message_expectations_on_nil
+    # allow(client).to receive(:messages)
+    # allow(client.messages).to receive(:list).and_return(["hello can i have a menu"])
```

Comment:
Metrics/LineLength: Line is too long. [86/80] (https://github.com/bbatsov/ruby-style-guide#80-character-limits)